docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -166,6 +166,11 @@ public class SantaClausService {
 You cannot use `UserTransaction` in a method having a transaction started by a `@Transactional` call.
 ====
 
+== Configuring the transaction timeout
+You can configure the default transaction timeout, the timeout that will applies to all transactions managed by the transaction manager, via the property `quarkus.narayana.default-transaction-timeout` (in seconds). The default value is unspecified in Quarkus, this means that the default of the transaction manager implementation is used.
+
+Currently, the transaction manager is Narayana that have a default of 60s.
```

Comment:
See my comment below about defining the default in Quarkus We could simplify the wording then.

Reply:
done